tests/all/build.rs

```diff
@@ -28,17 +28,17 @@ fn it_should_build_crates_in_a_workspace() {
             "Cargo.toml",
             r#"
                 [workspace]
-                members = ["blah"]
```

Comment:
i'm not sure i understand this change- can you explain? otherwise this looks good!